azurelinuxagent/ga/update.py

```diff
@@ -270,7 +293,7 @@ def _ensure_latest_agent(self):
         if len(manifests) == 0:
             msg = u"Incarnation {0} has no agent family {1} updates".format(etag, family)
             logger.info(msg)
-            add_event(u"WALA", message=msg)
+            add_event(AGENT_NAME, version=CURRENT_VERSION, message=msg)
```

Comment:
@boumenot I favor spewing as much data as we possibly can and then filtering at reporting. Again, for now, I've adopted the existing pattern. Let's work with this and change we get better data. Ok?